src/components/practice1.tsx

```diff
@@ -0,0 +1,18 @@
+import Button from './button';
+
+type ButtonType = 'button';
+export default function Practice1({name,styleb,type} : {name:string,styleb:string,type?:ButtonType})
```

Comment:
I have 2 points here.

1. Since this is the same type as Button's props, you should export the props in Button file and import to use it here.
It'll look something like:

```tsx

function Practice1({name,styleb,type} : ButtonProps)

```
2. In our project, we use **Named exports for components**, **Default exports for pages**

And we should opt for using **arrow functions** as this improves readability and follows [our React coding rules.](https://pj100.esa.io/posts/16606#%E3%82%A2%E3%83%AD%E3%83%BC%E9%96%A2%E6%95%B0%E3%82%92%E4%BD%BF%E7%94%A8%E3%81%99%E3%82%8B)

So please change this component into named exports like this:

```tsx
export const Practice1: React.FC<Practice1Props>: ({name,styleb,type})
```

Reply:
Done